cppConsole/.vscode/tasks.json

```diff
@@ -450,16 +450,16 @@
             ],
             "dependsOn": [
                 "validate-settings",
-                "validate-arch-x86",
+                "validate-arch-amd64",
                 "apply-torizon-packages",
                 "copy-docker-compose",
-                "pre-cleanup-x86",
-                "build-makedir-x86",
-                "build-debug-x86",
-                "build-container-torizon-debug-x86",
-                "push-container-torizon-debug-x86",
-                "pull-container-torizon-debug-x86",
-                "run-container-torizon-debug-x86",
+                "pre-cleanup-amd64",
```

Comment:
For example, here we are depending on `pre-cleanup-amd64` that does not exists

Reply:
This task was duplicated, so I just removed the duplicated one but it is still there. https://github.com/andreriesco/vscode-torizon-templates/blob/tie-716/assets/tasks/common.json#L591